core/src/main/java/com/linecorp/armeria/common/HttpHeadersBase.java

```diff
@@ -254,6 +263,33 @@ final void contentType(MediaType contentType) {
         set(HttpHeaderNames.CONTENT_TYPE, contentType.toString());
     }
 
+    @Nullable
+    @Override
+    public Locale selectLocale(Collection<Locale> supportedLocales) {
+        return acceptLanguages()
+                .stream()
+                .flatMap((it) -> Locale.filter(ImmutableList.of(it), supportedLocales).stream())
```

Comment:
nit: `(it)` -> `it`

Reply:
fixed. I'm starting to forget my Java :\